Adyen/UI/Form/Items/Address/FormFullAddressItem.swift

```diff
@@ -47,7 +47,7 @@ public final class FullFormAddressItem: FormValueItem<AddressInfo, AddressStyle>
         let item = FormLabelItem(text: localizedString(.billingAddressSectionTitle, localizationParameters),
                                  style: style.title)
         item.identifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: "title")
-        return item.withPadding(padding: .init(top: 8, left: 0, bottom: 0, right: 0))
+        return item.withPadding(padding: .init(top: 16, left: 16, bottom: 0, right: -16))
```

Comment:
Does this effect CardComponent?

Reply:
we're gonna rethink about this, because the `withPadding()` doesn't\t do what its supposed to do.